docs/windows-requirements.md

```diff
@@ -7,6 +7,7 @@
 
 | |
 | :- |
+| Boost ≥ 1.70 (latest stable recommended) |
```

Comment:
Does it work with 1.76

Reply:
now `Cmake` don't support 1.76 boost

Reply:
if apply commit https://gitlab.kitware.com/cmake/cmake/-/commit/79be37b94e7353ec056e6343ba293ce1b60627d4 - no problems

Reply:
Maybe better to not say the latest stable then but instead recommend 1.74?

Reply:
Hm, maybe better